src/mon/Monitor.cc

```diff
@@ -3424,7 +3424,15 @@ void Monitor::handle_command(MonOpRequestRef op)
 
   // validate user's permissions for requested command
   map<string,string> param_str_map;
-  _generate_command_map(cmdmap, param_str_map);
+
+  // Catch bad_cmd_get exception if _generate_command_map() throws it
+  try {
+    _generate_command_map(cmdmap, param_str_map);
+  }
+  catch(bad_cmd_get e) {
```

Comment:
```suggestion
  catch(bad_cmd_get& e) {
```
Otherwise, a warning pops up:
```
../src/mon/Monitor.cc: In member function ‘void Monitor::handle_command(MonOpRequestRef)’:
../src/mon/Monitor.cc:3432:21: warning: catching polymorphic type ‘struct ceph::common::bad_cmd_get’ by value [-Wcatch-value=]
   catch(bad_cmd_get e) {
                     ^
```

Reply:
Thanks for the review @ljflores , I've caught the exception by reference and pushed again.